lib/cb2/breaker.rb

```diff
@@ -4,6 +4,7 @@ class CB2::Breaker
   def initialize(options)
     @service  = options[:service] || "default"
     @strategy = initialize_strategy(options)
+    @whitelist = if options[:whitelist] then options[:whitelist] else [] end
```

Comment:
do you think it's worth considering adding validation code to make sure the values passed in to @whitelist are of type Class and inherit from StandardError?